arch/Alpha/AlphaDisassembler.c

```diff
@@ -0,0 +1,132 @@
+#ifdef CAPSTONE_HAS_ALPHA
+
+#include <stdio.h> // DEBUG
+#include <stdlib.h>
+#include <string.h>
+
+#include "../../utils.h"
+
+#include "../../MCFixedLenDisassembler.h"
+#include "../../MCDisassembler.h"
+
+static bool readInstruction32(const uint8_t *code, size_t code_len,
+			      uint32_t *insn)
+{
+	if (code_len < 4)
+		// insufficient data
+		return false;
+
+	// Encoded as a little-endian 32-bit word in the stream.
+	*insn = (code[0] << 0) | (code[1] << 8) | (code[2] << 16) |
```

Comment:
There is a function for this in `utils.c`

Reply:
changed to `readBytes` from `utils.c`